### ISSUE TITLE
Mockgen: generate interface mock in another package when method depends from another types of the same package

### ISSUE THREAD

**Issue by nawa** [golang/mock#152]
sample project structure is

```
gomocktst
├── mock_data
│   └── interface_mock.go
├── interface.go
├── interface_test.go
```

`interface.go` contains code:

```go
package gomocktst

type ReturnType string
type ParamType string

type Interface interface {
	Method(param ParamType) ReturnType
}
```

I want to generate mock of `Interface` in the `mock_data` package with next command

```
mockgen -source interface.go -package mock_data -destination mock_data/interface_mock.go
```

As a result I get inconsistent `MockInterface` where `ParamType` and `ReturnType` aren't imported. More detailed example here https://github.com/nawa/gomocktst/blob/master/mock_data/interface_mock.go#L36

Is it a bug, or I do something wrong?

**Comment by vail130** [comment]
I'm having this same issue. Would love some guidance on a workaround or status of a fix.

**Comment by nawa** [comment]
@vail130 The workaround is to fix it manually

**Comment by vail130** [comment]
I opened a PR with a fix, and then found another PR already opened with a fix. And then I switched to pegomock, which works :)

**Comment by balshetzer** [comment]
This has been fixed by #164.
